setup-certs.sh

```diff
@@ -15,7 +15,7 @@ SERVER_CSR="${LOCATION}/server.csr"
 echo $SERVER_KEY, $SERVER_CERT, $DHPARAM, $CA_KEY
 
 printf "# Create new dhparam. This may take a few minutes...\n"
-openssl dhparam -out $DHPARAM 2048
+openssl dhparam -out $DHPARAM 128
```

Comment:
Is this intentional, or just a temporary arrangement to make this run faster while testing?